snuba/datasets/events_processor_base.py

```diff
@@ -273,15 +283,15 @@ def extract_common(
 
         # Properties we get from the "data" dict, which is the actual event body.
 
-        received = _collapse_uint32(int(data["received"]))
+        received = _collapse_uint32(data["received"])
```

Comment:
The extra `int` here is probably a bug as `int(None)` fails, `_collapse_uint32(None)` returns `None`, and the subsequent code checks for `None`